services/openvpn/service/manager.go

```diff
@@ -299,10 +274,55 @@ func (m *Manager) ProvideConfig(_ string, sessionConfig json.RawMessage, conn *n
 			return nil, fmt.Errorf("could not proxy connection to OpenVPN server: %w", err)
 		}
 	}
-	return &session.ConfigParams{SessionServiceConfig: vpnConfig, TraversalParams: traversalParams}, nil
+
+	destroy := func() {
+		log.Info().Msgf("Cleaning up session %s", sessionID)
+
+		sessionClients := m.openvpnClients.GetSessionClients(session.ID(sessionID))
+		for clientID := range sessionClients {
+			if err := m.openvpnAuth.ClientKill(clientID); err != nil {
```

Comment:
Code doesn't compile, there is no such method ClientKill. Are you sure this PR is ready?

Reply:
I will update dependency, after is merged https://github.com/mysteriumnetwork/go-openvpn/pull/55